ibm/service/eventnotification/resource_ibm_en_subscription_email.go

```diff
@@ -0,0 +1,397 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package eventnotification
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	en "github.com/IBM/event-notifications-go-admin-sdk/eventnotificationsv1"
+	"github.com/IBM/go-sdk-core/v5/core"
+)
+
+func ResourceIBMEnEmailSubscription() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMEnEmailSubscriptionCreate,
+		ReadContext:   resourceIBMEnEmailSubscriptionRead,
+		UpdateContext: resourceIBMEnEmailSubscriptionUpdate,
+		DeleteContext: resourceIBMEnEmailSubscriptionDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"instance_guid": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Unique identifier for IBM Cloud Event Notifications instance.",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Subscription name.",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Subscription description.",
+			},
+			"destination_id": {
```

Comment:
In all resources we don't have support for few attributes  like destination_id and topic_id here can we mark them as Forcenew...(holds for all resources)

Reply:
I have updated the subscription resources changes for this. In case of destination resources I haven't made changes since I don't want any of given attribute alteration to create a new destination resource. If there is any specific requirement please feel free to point it out.